components/automate-ui/package.json

```diff
@@ -15,10 +15,10 @@
     "copy-ui-lib:fonts": "cp -r ../chef-ui-library/dist/collection/assets/fonts/ src/assets/fonts",
     "build": "ng build",
     "build:prod": "npm run build -- --prod",
-    "test": "scripts/unit-tests.sh",
+    "test": "scripts/build.sh 'ng test --watch=false --code-coverage --source-map=false --configuration=no_auth' 'WARN:'",
```

Comment:
Nice side effect is that the command is now surfaced back in package.json instead of buried in the script.